src/cloudfront/index.spec.ts

```diff
@@ -387,7 +387,7 @@ describe("cloudfront", () => {
       },
       { shouldBlockBucketPublicAccess: false },
     ])(
-      "should not update the distribution if the right origin is already associated",
+      `should not update the distribution if the right origin is already associated %p`,
```

Comment:
NIT: Maybe add "shouldBlockBucketPublicAccess: " before `%p`, might be hard to understand the test like this

Reply:
I didn't know either but `%p` was suggested by copilot and print nicely the param:
<img width="1047" alt="image" src="https://github.com/lalalilo/aws-spa/assets/19348797/ec0bd96e-9f87-4d35-8664-a156b0d07cb0">